README.md

```diff
@@ -171,10 +256,22 @@ Head on over to [./_examples](./_examples) to get familiar with **callbacks**, *
 
 ---
 
-## Workflow's RunState
-RunState is the state of a Run and can only exist in one state at any given time. RunState is a
- finite state machine and allows for control over the Run. A Run is every instance of
- a triggered workflow.
+## What is a workflow Run
+
+When a **Workflow** is triggered it creates an individual workflow instance called a Run. This is represented as workflow.Run in
+**Workflow**. Each run has a lifecycle which is a finite set of states - commonly
+referred to as Finite State Machine. Each
+ workflow Run has the following of states (called RunState in **Workflow**):
+
+1. Initiated
+2. Running
+3. Paused
+4. Completed
+5. Cancelled
+6. Data Deleted
+7. Requested Data Deleted
```

Comment:
Is this in the order of the enum? Feels slightly easier to follow if `Requested Data Deleted` is before `Data Deleted` 😄

Reply:
It was added after and these are the assigned values. Maybe I should make that more clear.